selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/user/TestUserFactory.java

```diff
@@ -17,7 +17,13 @@
  * @author Dmytro Nochevnov
  */
 public interface TestUserFactory {
-  TestUser create(@Assisted("email") String email);
 
-  TestUser create(@Assisted("email") String email, @Assisted("password") String password);
+  /** Creates new test user with generated e-mail and password */
+  TestUserImpl create();
```

Comment:
What is the propose of this method? 
It looks as redundant so as simple `@Inject TestUser test user;` could be used instead of testUserFactory.create()

Reply:
This method needs in case when in tests there need to be some user with generated email and password. So for codenvy I planed to bind the Test user to implementation wich call this method.

Reply:
I see. But we don't need assisted injection, that is TestUserFactory, for creation an instance from constructor without **@Assisted** parameters at all. 
@Inject annotation is able to provide required instance of TestUser.

Reply:
removed method + changed annotation